scripts/delete_resources.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/bash
 
 #Get resource name from tfvarslocal && change name to make more sense in this context
-PRODUCT_NAME=$(grep ENV_PRODUCT <../config/config.yaml | cut -d: -f2 | tr -d ' "')
+PRODUCT_NAME=$(grep ENV_PRODUCT <../config/product.yaml | cut -d: -f2 | tr -d ' "')
 RESOURCE_NAME=$(grep resource_name <../config/"$PRODUCT_NAME".tfvars | cut -d= -f2 | tr -d ' "')
+
+if [[ -z "$RESOURCE_NAME" ]]; then
+  echo "No file or directory found"
+  exit 1
+fi
+
+if  [[ "$RESOURCE_NAME" != "$PRODUCT_NAME".tfvars ]]; then
+  echo "Wrong tf.vars file found"
```

Comment:
nit: Fix typo - `Wrong tfvars file found` Also suggest to add the RESOURCE_NAME variable on failures/errors